### ISSUE TITLE
World's not loading properly

### ISSUE THREAD

**Issue by Starwolf16** [sp614x/optifine#3752]
Hello,
All of the worlds that I had created before installing optifine will not load properly, I get stuck in the world and there aren't any blocks that load. However, if I run Minecraft without optifine, the world loads just fine. Also, if I create a new world while using optifine, the world loads fine. 

Using Optifine 1.15.2 Pre 13

**Comment by Jumpybug214** [comment]
i use the same version as you, yet ive never had this issue. maybe its your os/java version?

**Comment by CHAREDot9** [comment]
I had a similar issue but it happened in a different way. I created a single player world, Installed OptiFine 1.15.2 HD U G1 pre13 and played with it for about a week with no issues. I was placing blocks in the end and the game stopped letting me place blocks so i exited to the menu and rejoined and there were 0 chunks loaded. The world still works fine without optifine. The game still loads chunks in my other worlds with the same optifine login

**Comment by AndyPandy05** [comment]
I am having the same problem, i was able to fix it yesterday by changing render dist, but when i try to fix it again the same way now it doesn't work. I am on optifine 1.15.2 HD U G1 pre13

**Comment by KawaiiZapic** [comment]
I have same problem too.
This problem exists for a long time(1.15.2 Pre x - Pre 13).

My problem happens in this way:
I often disconnected from server for network problems or personal reasons.
And then the problem will appear if I disconnected many times.
When I rejoin into server,no block is visible,but I can see the outline of block.
Then I give world an update(open door,place blocks,etc),blocks in this chunk will be visible,only the chunk that I give a update.
This problem will be fixed if I restart the game.
And looooooooooooooop,start game, disconnected,rejoin, problem appear,close game,start game.

**Comment by zabrake-1** [comment]
This issue is happening for me too. Specifically for version: OptiFine 1.15.2 HD U G1 pre13. The world only loads (by default) a small square of the initial spawn world and then fails to render anything further. There is a photo attached to the below reddit post for reference.

The issue does not occur on older versions of OptiFine but is frustrating as it's unusable on the current 1.15.2 version - hopefully the dev might see this and rectify in the next edition.

https://www.reddit.com/r/Worldpainter/comments/g15w2p/help_any_export_of_worldpainter_to_1152_really/

**Comment by MadLad-cyber** [comment]
I had this issues on one of my worlds so I set about troubleshooting it;

I managed to actually fix it by lowering my render distance down to minimum for initial load

**Comment by GucciToucan** [comment]
I have the solution to this problem. Just lower the render distance to 2 then raise it back up to your main render distance and the problem with rendering the world should be fixed.

**Comment by stephenlb** [comment]
Same issue.  World just stops loading after you get to a point on the map.  It's just blank.  The world is flat.

**Comment by stephenlb** [comment]
![Screen Shot 2020-05-08 at 7 09 09 PM](https://user-images.githubusercontent.com/45214/81461280-9ec73f00-915f-11ea-84b7-5c48a812a9ba.png)
![Screen Shot 2020-05-08 at 7 09 22 PM](https://user-images.githubusercontent.com/45214/81461284-a25ac600-915f-11ea-9128-c71afa63c9a3.png)
![Screen Shot 2020-05-08 at 7 09 27 PM](https://user-images.githubusercontent.com/45214/81461285-a2f35c80-915f-11ea-9670-c3be02cb347a.png)
![Screen Shot 2020-05-08 at 7 09 38 PM](https://user-images.githubusercontent.com/45214/81461286-a38bf300-915f-11ea-8782-fd9a7e84585c.png)
![Screen Shot 2020-05-08 at 7 09 51 PM](https://user-images.githubusercontent.com/45214/81461287-a4248980-915f-11ea-858f-734f586211cd.png)
![Screen Shot 2020-05-08 at 7 09 54 PM](https://user-images.githubusercontent.com/45214/81461288-a4248980-915f-11ea-8831-3a5e27c3867d.png)
![Screen Shot 2020-05-08 at 7 09 57 PM](https://user-images.githubusercontent.com/45214/81461289-a4bd2000-915f-11ea-9a24-284b43c1501b.png)

**Comment by stephenlb** [comment]
Trying the min-distance render that @MadLad-cyber mentioned.

**Comment by stephenlb** [comment]
Confirmed.  @MadLad-cyber was correct.  Reducing the draw distance to minimum on first load did the trick!  Increased draw distance to maximum after and all was good.  Thank you.

**Comment by clementlize** [comment]
> I had this issues on one of my worlds so I set about troubleshooting it;
> 
> I managed to actually fix it by lowering my render distance down to minimum for initial load

This is THE answer I've been looking for, thank you for sharing!

**Comment by sp614x** [comment]
Can someone post a link to a (small) zipped world save which can't be opened with OF 1.15?

**Comment by clementlize** [comment]
> Can someone post a link to a (small) zipped world save which can't be opened with OF 1.15?

I made some new tests. 
 - I tried to load 3 "complex" worlds, meaning worlds I've explored and in which I built things. These 3 won't load with a Render Distance (RD) of 48 chunks. But they load perfectly with a RD of 36 or less.
 - I created a new world and it loads well with a RD of 48.

Here is one of the 3 "complex" worlds: [link](https://mega.nz/file/35ojUKAC#Ys6Ggzj_hDqO8ZLJX850VLpv1GhJd9u8WeWb0cS51No)
Here is the new world: [link](https://mega.nz/file/Gop3mApZ#kGbezL4cAyPfSCDQXP0Qqut9Yqr8etmYFcQGEH0Krig)

**Comment by realjackasterisk** [comment]
Possible duplicate of #3365. Leaving open for now since this has more information and is a bit more active.

**Comment by KhrisRiddle** [comment]
I'm in optifine 1.15.2 HD U G1 pre30. I was having the same problems when creating and loading worlds in single player only when my render distance is greater than 31. But in multiplayer it works perfectly fine no matter the render distance. I have also experienced where you get stuck when you tp to cords and such, but again only in single player.

**Comment by ParmesanPizza** [comment]
THis is also happening in the newer versions of optfine (in 1.16.1).

**Comment by OverSeerYT** [comment]
have you already fixed it? I am running 1.16.1 too and this happens.

**Comment by ParmesanPizza** [comment]
no. I just have to run 1.16.1 or 1.16 2, which severely impacts performance...

**Comment by OverSeerYT** [comment]
the render distance worked for me tho..

**Comment by OverSeerYT** [comment]
i use ForgeOptifine

**Comment by cmexp** [comment]
I can confirm that this happens on 1.16.2. Reloading chunks does not help. Decreasing the render distance to 2 upon load does reset the area loaded and fixes the problem for that area, but it still happens when I max the render distance again, just with a different area of the map. I have no additional mods installed apart from a shader pack, and it still happens with that disabled. Works fine in vanilla.

**Comment by cihan-design** [comment]
Same thing is happening to me right now, i have a 1.16.2 vanilla world with data packs, it works perfectly fine in vanilla but doesn't work on optifine 1.16.2 no matter what i do, here is what i've tried so far:

- lowered the render distance and reloaded the chunks f3+a
- deleted all the datapacks and tried starting it again
- opened a new world, carried these files to the world file that i can't open with optifine;
level.dat
level.dat_old
session.lock

none of these worked but the last one partially worked, it actually loaded a chunk in a different location. (my base coordinates are approximately -3600 -1400 and when i did the last one it loaded -120 164) but couldn't work any other chunks

![2020-09-19_14 53 17](https://user-images.githubusercontent.com/71553536/93667038-7cd5b580-fa8b-11ea-97cf-3816e3bca7f8.png) the result of last option

![2020-09-19_14 16 14](https://user-images.githubusercontent.com/71553536/93667064-a7c00980-fa8b-11ea-990f-69fb438ee6d5.png) my base but chunks are not loading

**Comment by WoodWorker54** [comment]
Can confirm that it is still an issue in 1.16.2 v.3. Everything will work perfectly in vanilla and no optifine and also in multiplayer with optifine, but for some reason on single player with optifine the world will just not load (takes about 1 minute to load anything). Once everything is loaded then it's all fine. I've tried all of the "fixes" listed here in the comments but none of them have worked. Minimizing the render distance to the minimum for example did nothing, took just as long to load in.

**Comment by P14S0P** [comment]
if someone still have this problem, disable Connected Textures.
